hw/xbox/dsp/dsp_cpu.c

```diff
@@ -159,6 +159,11 @@ static bool match_MMMRRR(uint32_t op)
     return true;
 }
 
+static OpcodeEntry* search_opcode(uint32_t op);
+
+/*nonparallel opcodes only occupy the lower 20bits*/
+OpcodeEntry *OpcodeEntryArray[0xFFFFF+1];
```

Comment:
Given how `OpcodeEntryArray` is used, and the other symbol names, it might be clearer to call it `nonparallel_opcode_lookup_array`.